content/release.rst

```diff
@@ -197,3 +210,78 @@ Sample release notes
     * Fix an internal server error when FTP client requests
       an unknown command. [#160][ftp][ftps]
 
+
+Version Management
+==================
+
+Chevah release versions are based on MAJOR.MINOR.PATCH
+`Semantic Versioning specification <http://semver.org/>`_.
+
+A MAJOR version is released to introduce new features,
+remove functionalities which have become obsolete, or add features not
+compatible with previous versions.
+
+MINOR versions are released based on a rolling update development model at
+intervals varying between 30 to 60 days.
+The goal is to have functionalities and defect fixes available to customers as
+soon as possible.
+Each release has a certain overhead, and the overhead should be minimize by
```

Comment:
Comma not required here because the introductory clause is not dependent on the main clause. More at https://trac.chevah.com:10443/wiki/Development/EnglishForRomanians#comma-before-conjunctions

Also _should be minimize_ -> _should be minimize**d**_.